specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1608,6 +1603,18 @@
       ]
     },
     "KeyExportParameters": {
+      "properties": {
+        "kek": {
+          "$ref": "#/definitions/JsonWebKey",
+          "description": "The export key encryption Json web key. This key MUST be a RSA key that supports encryption."
```

Comment:
Nit: JSON
```suggestion
          "description": "The export key encryption JSON web key. This key MUST be a RSA key that supports encryption."
```

Reply:
I hated that, but it was coped over and I was not sure whether the casing was a "standard" for the REST API docs.